Changes

```diff
@@ -466,7 +466,7 @@ Devel::Cover history
  - Fix gcov2perl to work with large numbers (Thomas Dorner) (RT-45028)
  - Fix "gcov -l" include files (Thomas Dorner) (RT-44864)
  - Test against 5.12.1
- - Gross workaround for for regexp evals (Florian Ragwitz)
+ - Gross workaround for regexp evals (Florian Ragwitz)
```

Comment:
another not precisely spelling fix, but spell checking has handled this for decades.